interface/build.config.js

```diff
@@ -64,6 +64,12 @@ module.exports = {
    */
   vendor_files: {
     js: [
+      // For some unknown reason(s), jquery and jquery-ui library files MUST be
+      // put at the beginning of this array; Otherwise the ui-slider wouldn't
+      // show up on the web UI, and we got the following error in the console:
```

Comment:
Angular is compatible with jQuery. Part of it is implemented with a stripped-down jQuery they call jQuery-lite, and, in fact, Angular itself will automatically switch to using the jQuery you supply instead of jQuery-lite. I do not count this as a feature.

The possibility of having Angular itself change behavior with a change in jQuery is unsettling. I suspect this "adaptation" to the supplied jQuery is responsible for the sensitivity to JavaScript module loading order. Angular and jQuery will confuse each other unless they are loaded together exactly the right way.